tests/test_models.py

```
@@ -111,3 +115,372 @@ def test_get_content(self, app):
+            assert sample_user_skill.user.id == sample_user.id
+            assert sample_user_skill.skills.id == sample_skill.id
+
+
+class TestModelRelationships:
+    """Test complex model relationships and cascading"""
+
+    def test_job_cascade_delete(self, app, test_data_builder):
+        """Test that deleting a job cascades to related models"""
+        with app.app_context():
+            # Create job with skills and logs
+            job_data = test_data_builder.create_job_with_skills_and_logs(
```

Ruff (0.13.1)

Local variable `job_data` is assigned to but never used

Remove assignment to unused variable `job_data`

(F841)